.gitignore

```diff
@@ -7,7 +7,8 @@ report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 .yarn-integrity
 
 # Mock data
```

Comment:
The comment `Mock data` applies only to the folder `dataOverlay`, not `neo4j`.

Reply:
Good catch! Fixed in: https://github.com/MetabolicAtlas/data-generation/pull/20/commits/2de5949bcc572f52d69544a8752f8e01af255587